Control/move_to_pose/move_to_pose.py

```diff
@@ -63,6 +67,12 @@ def move_to_pose(x_start, y_start, theta_start, x_goal, y_goal, theta_goal):
         if alpha > np.pi / 2 or alpha < -np.pi / 2:
             v = -v
 
+        if abs(v) > _MAX_LINEAR_SPEED:
+            v = v/abs(v) * _MAX_LINEAR_SPEED 
+
+        if abs(w) > _MAX_ANGULAR_SPEED:
+            w = w/abs(w) * _MAX_ANGULAR_SPEED
```

Comment:
Could you please add a test for your addition? https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_move_to_pose.py

Reply:
Should I add another file with the name like 'test_move_to_pose_with_speed_limit.py'? Doesn't the current test do the same thing?

Reply:
Adding a new test in the file is OK. You can change the two specifications by changing the global variables, then the control results should be changed. Checking it in the test is OK.

Reply:
I included the test_2 in the original test file.